include/CLoop.h

```diff
@@ -44,6 +44,7 @@ class CLoop {
    void     Loop(float lumFactor, int z_sample, std::string key, const CLoopConfig& config);
    Bool_t   Notify();
    void     Show(Long64_t entry = -1);
+   int GetNEntries();
```

Comment:
Sorry I don't understand why you need this? Also one has to be careful, if you are getting the number of entries, this could be in principle a big number. So I would make this a `long` or something that allows more numbers.

Reply:
I used this very briefly to check which files were empty when I thought that was an issue - it's obsolete now, so I'll remove it

Reply:
Did you also remove the references to this code in the `/python` folder?

Reply:
Have removed them now